doc/source/user_guide/io.rst

```diff
@@ -3833,7 +3833,7 @@ OpenDocument Spreadsheets
 The io methods for `Excel files`_ also support reading and writing OpenDocument spreadsheets
 using the `odfpy <https://pypi.org/project/odfpy/>`__ module. The semantics and features for reading and writing
 OpenDocument spreadsheets match what can be done for `Excel files`_ using
-``engine='odf'``.
+``engine='odf'``. ``pip install odfpy`` may be required, if is not already installed.
```

Comment:
It's sufficient if you add a comment that users have to install the optional dependency oddly, pip install is not necessary.

Reply:
I replaced it with a comment, and this time?